libs/R-TypeLogic/EntityManipulation/CreateEntitiesFunctions/CreateWritableButton.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+** EPITECH PROJECT, 2022
+** R-Type
+** File description:
+** CreateWritableButton
+*/
+
+#ifndef CREATEWRITABLEBUTTON_HPP_
+#define CREATEWRITABLEBUTTON_HPP_
+
+#include "World/World.hpp"
+#include "R-TypeLogic/EntityManipulation/ButtonManipulation/Components/ActionName.hpp"
+#include "R-TypeLogic/EntityManipulation/ButtonManipulation/Components/DisplayState.hpp"
+#include "R-TypeLogic/Global/Components/ButtonComponent.hpp"
+
+namespace ecs
```

Comment:
Is it a good namespace "ecs" ?
is it "R-TypeLogic::EntityManipulation::CreateEntitiesFunctions" ?

Reply:
Refer to discord conversation